src/ui/widget-forms/HandpickedContentListConfigFormContainer.js

```diff
@@ -12,8 +12,10 @@ import { getSearchPagesRaw } from 'state/pages/selectors';
 import { getActiveLanguages } from 'state/languages/selectors';
 import { sendPutWidgetConfig } from 'state/page-config/actions';
 import { ROUTE_APP_BUILDER_PAGE_CONFIG } from 'app-init/routes';
-import { formValueSelector, reduxForm } from 'redux-form';
+import { formValueSelector, reduxForm, submit } from 'redux-form';
 import { MULTIPLE_CONTENTS_WIDGET_CONFIG_ID } from 'ui/widget-forms/const';
```

Comment:
for CMS widget forms please let's use the same naming for component name and identifier (e.g. `HandpickedContentListConfigFormContainer` vs  MULTIPLE_CONTENTS_WIDGET_CONFIG_ID).